{{cookiecutter.profile_name}}/config.yaml

```diff
@@ -1,11 +1,11 @@
 latency-wait: "{{cookiecutter.latency_wait}}"
-jobscript: "lsf-jobscript.sh"
+jobscript: "lsf_jobscript.sh"
 use-conda: "{{cookiecutter.use_conda}}"
 use-singularity: "{{cookiecutter.use_singularity}}"
 printshellcmds: "{{cookiecutter.print_shell_commands}}"
 restart-times: "{{cookiecutter.restart_times}}"
 jobs: "{{cookiecutter.jobs}}"
-cluster: "lsf-submit.py"
-cluster-status: "lsf-status.py"
+cluster: "lsf_submit.py"
+cluster-status: "lsf_status.py"
```

Comment:
Why did you rename these files? I believe this is why this PR has a conflict.

Reply:
We can't easily import files having a `-` in the filename in the test cases to test them. I mean, there is a way around it: https://stackoverflow.com/questions/8350853/how-to-import-module-when-module-name-has-a-dash-or-hyphen-in-it , but it seemed easier to simply rename the file...

Reply:
Ah, I didn't realise this. You're right, renaming is the better solution.